### ISSUE TITLE
cardiac arrest

### ISSUE THREAD

**Issue by anna-anagnostop** [obophenotype/mammalian-phenotype-ontology#4327]
Consider adding:
cardiac arrest
DEF: a sudden cessation of cardiac activity due to an irregular heart rhythm resulting in hemodynamic collapse; key signs include loss of consciousness, loss of normal breathing, and absence of a cardiac tone or pulse
SYN: sudden cardiac arrest (narrow)
SOURCE: https://www.ncbi.nlm.nih.gov/books/NBK507854/, https://www.mayoclinic.org/diseases-conditions/sudden-cardiac-arrest/symptoms-causes/syc-20350634
Annotations from PMID: 10949023 ([J:63879](https://www.informatics.jax.org/reference/J:63879))

**Comment by anna-anagnostop** [comment]
In PMID: 10949023 ([J:63879](https://www.informatics.jax.org/reference/J:63879)), cardiac arrest occurs at E10.5 and is currently annotated to embryonic lethality during organogenesis, complete penetrance MP:0011098.

HPO places Sudden cardiac death under Sudden death which is-a [Constitutional symptom](https://www.informatics.jax.org/vocab/hp_ontology/HP:0025142).

Do we need separate terms for cardiac arrest in embryos versus adults in the MP? If we place sudden cardiac death under premature death (via sudden death), then we cannot have sudden cardia death under cardiac arrest, given that annotations from J:63879 refer to embryos.

**Comment by anna-anagnostop** [comment]
How does cardiac arrest relate to asystole?

See https://www.ncbi.nlm.nih.gov/books/NBK430866/ for a definition of Asystole. 
Looking at current MGI annotations to absent heartbeat (synonym asystole) it looks like we may need to use "sudden cardiac arrest" as the primary label, define it as an abrupt cessation of cardiac function (similar to the HPO term) and reserve this term for adulthood. Need to do a bit more digging.
